en/docs/getting-started/overview.md

```diff
@@ -1,99 +1,113 @@
 # Overview
 
-WSO2 API Manager is a fully open source solution for end to end API Management in the cloud, on-prem or in hybrid environments. It comes with an [Apache Software License Version 2.0](http://www.apache.org/licenses/LICENSE-2.0) which makes it **free to use**. It allows API developers to design publish and manage the lifecycle of APIs and API product managers to create API products from one or more APIs. It hosts an application developer portal which helps in building and managing a developer community for your APIs. Its cloud native API gateway is used for securing, routing, controlling and monitoring your API traffic in a scalable manner.
+WSO2 API Manager is a fully open-source solution for end to end API Management in the cloud, on-prem, or in hybrid environments. It comes with an [Apache Software License Version 2.0](http://www.apache.org/licenses/LICENSE-2.0) which makes it **free to use**. It allows API developers to design publish and manage the lifecycle of APIs and API product managers to create API products from one or more APIs. It hosts an application developer portal which helps in building and managing a developer community for your APIs. Its cloud-native API gateway is used for securing, routing, controlling, and monitoring your API traffic in a scalable manner.
 
-## **What is the WSO2 API Manager** ?
+## **What is WSO2 API Manager?**
 
-WSO2 API Manager is the only 100% open source product that addresses all aspects of API management including API development, productization, consumption, security, rate limitation, monetization and analytics.
+WSO2 API Manager is the only 100% open-source product that addresses all aspects of API management including API development, productization, consumption, security, rate limitation, monetization, and analytics.
 
-![]({{base_path}}/assets/attachments/103327648/103346653.png)
-In brief, the WSO2 API Manager consists of 6 main components, the [API Publisher,](#api-publisher) the [API developer Portal,](#developer-portal) [API Gateway](#api-gateway) (now comes as a [API Microgateway](https://docs.wso2.com/display/MG300/API+Microgateway+Documentation) as well ), [Key Manager,](#key-manager) [Traffic Manager](#traffic-manager) and the [API Analytics](#analytics). For more information on these components see [Basic Architecture and Key Components]().
+![WSO2 API Manager Overview]({{base_path}}/assets/attachments/103327648/103346653.png)
```

Comment:
please rename the image and add it to the correct location and use it.

Reply:
Appreciate if you can point share any practices that have to be followed on arranging and managing the assets. Based on that, will change the locations and refactor the artifacts :)

Reply:
we use the same directory structure used for the docs in the assets as well.